frontend/src/component/admin/billing/BillingDashboard/BillingPlan/useOverageCost.ts

```diff
@@ -1,26 +1,16 @@
 import { endOfMonth, format, startOfMonth } from 'date-fns';
-import {
-    useInstanceTrafficMetrics,
-    useTrafficSearch,
-} from 'hooks/api/getters/useInstanceTrafficMetrics/useInstanceTrafficMetrics';
+import { useTrafficSearch } from 'hooks/api/getters/useInstanceTrafficMetrics/useInstanceTrafficMetrics';
 import { useMemo } from 'react';
 import {
     calculateOverageCost,
     calculateTotalUsage,
 } from 'utils/traffic-calculations';
 import { BILLING_TRAFFIC_BUNDLE_PRICE } from './BillingPlan';
-import { useUiFlag } from 'hooks/useUiFlag';
-import { useTrafficDataEstimation } from 'hooks/useTrafficData';
 
-export const useOverageCost = (includedTraffic: number) => {
-    if (useUiFlag('dataUsageMultiMonthView')) {
-        return useNewOverageCostCalculation(includedTraffic);
-    } else {
-        return useOldOverageCostCalculation(includedTraffic);
-    }
-};
+export const useOverageCost = (includedTraffic: number) =>
+    useOverageCostCalculation(includedTraffic);
```

Comment:
is this necessary? couldn't we just export the `useOverageCostCalculation` at this point?

Reply:
Or at least just inline the execution, yeh. Good point; will fix